static/app/components/segmentedControl.tsx

```diff
@@ -245,7 +246,7 @@ const SegmentSelectionIndicator = styled(motion.div)<{priority: Priority}>`
     p.priority === 'primary'
       ? p.theme.borderRadius
       : `calc(${p.theme.borderRadius} - 1px)`};
-  box-shadow: 0 0 2px rgba(43, 34, 51, 0.16);
+  box-shadow: 0 0 2px rgba(43, 34, 51, 0.32);
```

Comment:
Heavier shadow to better separate the selected segment from the rest.

**Before ——**
<img width="218" alt="Screenshot 2023-02-02 at 10 32 19 AM" src="https://user-images.githubusercontent.com/44172267/216417498-c0d92197-40ea-42ab-aaee-17aeafc42fd2.png">

**After ——**
<img width="218" alt="Screenshot 2023-02-02 at 10 32 02 AM" src="https://user-images.githubusercontent.com/44172267/216417380-fb4fc4e9-2589-44e8-8792-b0ab23c2ae14.png">